### ISSUE TITLE
The store gets reloaded twice

### ISSUE THREAD

**Issue by NorbNorb** [lhunath/UbiquityStoreManager#52]
The following code block taken from the current revision reloads the store twice (code summary):

```
- (void)setCloudEnabled:(BOOL)enabled {
...
    [[NSUserDefaults standardUserDefaults] setBool:enabled forKey:USMCloudEnabledKey];
    [self reloadStore];
}

- (void)userDefaultsDidChange:(NSNotification *)note {
    if (self.cloudWasEnabled != self.cloudEnabled)
        [self reloadStore];
}
```

Is this intended?

**Comment by lhunath** [comment]
No, that is unintended.  If the store is reloaded twice there, the top reloadStore should be removed.

**Comment by NorbNorb** [comment]
Thanks for your reply! Will you fix this in your source?

**Comment by lhunath** [comment]
Should be fixed by ef9aa1c
